src/common/utils/tools.js

```diff
@@ -147,3 +147,9 @@ export const errLogger = (err, msg) => {
     const errMsg = `${msg} - Error: ${errStr}`;
     console.warn(errMsg);
 };
+
+export const isBinaryDomain = !(
+    document.location.hostname !== 'localhost' &&
+    document.location.hostname !== 'bot.binary.com' &&
+    !document.location.hostname.includes('binary-bot-git-fork')
+);
```

Comment:
Why do we need container '!( ...)'? Why can't we specify the right variables at once ?

Reply:
I used this approach which once was approved by Yashim as part of branding removal for other projects. In this case we can change it. Please check again 🙏